### PR TITLE
Write analysis for concept exercise `language-list`

### LINKED ISSUE #148: Write analysis for concept exercise `language-list`
Add a check that the `Enum` module is not used in the solution. Something like this sound be enough: https://github.com/exercism/elixir-analyzer/blob/972446957878055dd503170f255688bef8361a93/lib/elixir_analyzer/test_suite/bird_count.ex#L8-L12

This exercise teaches lists and the simple functions from the `Kernel` module and the `List` for working with lists.

### PR DESCRIPTION
**Pull request by natanaelsirqueira**
Closes #148. Related PR: https://github.com/exercism/website-copy/pull/2074.

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -48,6 +48,9 @@ config :elixir_analyzer,
     "high-score" => %{
       analyzer_module: ElixirAnalyzer.TestSuite.HighScore
     },
+    "language_list" => %{
```
Comment (jiegillet) [inline config/config.exs:51]: This needs to be the slug
```suggestion
    "language-list" => %{
```

**Review by jiegillet — COMMENTED** [review]
Looks good, thank you for implementing!
You only need to change one character!

**Comment by natanaelsirqueira** [comment]
@jiegillet Thanks. Fixed!

**Review by jiegillet — APPROVED** [review]
Nice one, thank you :)

**Review by angelikatyborska — APPROVED** [review]
Perfect!

**Pull request merged**